pkg/util/ip_reservation.go

```diff
@@ -0,0 +1,182 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"fmt"
+	"net"
+	"sync"
+)
+
+const (
+	incrementStep29IPRange = 8
+	reservationRange       = "/29"
+	byteMax                = 255
+	maximumCIDRSize        = 29
+	// Mask to check if cidr is aligned
+	ipRangeMask = "255.255.255.248"
```

Comment:
We have 3 constants that all represent the "/29" mask.  Is there a way we can consolidate to 1 constant and derive the other forms when needed?